Ryujinx.HLE/HOS/Services/Nv/NvDrvServices/NvHostChannel/NvHostChannelDeviceFile.cs

```diff
@@ -0,0 +1,352 @@
+﻿using ARMeilleure.Memory;
+using Ryujinx.Common.Logging;
+using Ryujinx.Graphics;
+using Ryujinx.Graphics.Memory;
+using Ryujinx.HLE.HOS.Services.Nv.NvDrvServices.NvHostAsGpu;
+using Ryujinx.HLE.HOS.Services.Nv.NvDrvServices.NvHostChannel.Types;
+using Ryujinx.HLE.HOS.Services.Nv.NvDrvServices.NvMap;
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace Ryujinx.HLE.HOS.Services.Nv.NvDrvServices.NvHostChannel
+{
+    class NvHostChannelDeviceFile : NvDeviceFile
+    {
+        private uint           _timeout;
+        private uint           _submitTimeout;
+        private uint           _timeslice;
+        private NvGpu          _gpu;
+        private IMemoryManager _memory;
+
+        public NvHostChannelDeviceFile(ServiceCtx context) : base(context)
+        {
+            _gpu            = context.Device.Gpu;
+            _memory         = context.Memory;
+            _timeout        = 3000;
+            _submitTimeout  = 0;
```

Comment:
```suggestion
            _submitTimeout = 0;
```
and same on other vars